### DISCUSSION TITLE
disable-zoom still not working in v1.4.1

### DISCUSSION THREAD

**Discussion by aresrioja10** [google/model-viewer#1964] [Q&A]
### Description

I've upgraded the @google/model-viewer package to 1.4.1 and the disable-zoom property is not working.

#### Live Demo
This is my React code:

```
import '@google/model-viewer';

<model-viewer id={modelId} src={modelSrc} disable-zoom={true} />
```
### Version

- model-viewer: v1.4.1

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [X] MacOS
- [ ] Windows

**Comment by alexdaube** [comment]
The problem lies on how react handles attribute conversion when used with a web component if I remember correctly. 

You can do something of that sort as a workaround and it should work. At least it does for me. 

```
const attributes = { 
 'disable-zoom': shouldDisableZoom ? true : undefined}
};

<model-viewer {...attributes}>...</model-viewer>
```

**Comment by aresrioja10** [comment]
> The problem lies on how react handles attribute conversion when used with a web component if I remember correctly.
> 
> You can do something of that sort as a workaround and it should work. At least it does for me.
> 
> ```
> const attributes = { 
>  'disable-zoom': shouldDisableZoom ? true : undefined}
> };
> 
> <model-viewer {...attributes}>...</model-viewer>
> ```

It works!
